server/api.js

```diff
@@ -19,74 +16,129 @@ const router = new Router();
 // 	});
 // });
 
-router.get("/", (_, res, next) => {
-	const client = getClient();
-	client.connect((err) => {
-		if (err) {
-			return next(err);
-		}
-		const db = client.db("feedback-tracker");
-		const collection = db.collection("students");
-		// const searchObject = {};
+const client = getClient();
+
+client.connect(function () {
+
+	const db = client.db("feedback-tracker");
+
+	router.get("/students", (_, res) => {
+		const collection = db.collection("StudentDenormalizedData");
+		console.log("collection", collection);
+
 		collection
 			.find()
-			.toArray(function(error, students) {
-				res.send(error || students);
-				// client.close();
```

Comment:
I would change the name of the collection and use students instead. However I understand this as well.

Reply:
It is because we already have a collection called students, we can change it once we delete the old one.